src/seccomp/src/lib.rs

```diff
@@ -1050,6 +1042,14 @@ impl SeccompFilter {
         }
         self
     }
+
+    /// Creates an empty SeccompFilter which allows everything
```

Comment:
nit
```suggestion
    /// Create an empty `SeccompFilter` which allows everything.
```

Reply:
I've followed the tone of the rest of the comments in this crate, which use the conditional tense: builds, replaces, returns.